app/javascript/packs/component-definitions-common.js

```diff
@@ -58,6 +62,7 @@ ManageIQ.component.addReact('MiqAboutModal', MiqAboutModal);
 ManageIQ.component.addReact('OptimizationList', OptimizationList);
 ManageIQ.component.addReact('OpsTenantForm', OpsTenantForm);
 ManageIQ.component.addReact('OrcherstrationTemplateForm', OrcherstrationTemplateForm);
+ManageIQ.component.addReact('Quadicon', Quadicon);
```

Comment:
This should probably come after `PxeServersForm` (if we want to do this alphabetically).